Error log in test_list_image_topics

In [TEST_F(TestListImageTopics, TestOne)](https://github.com/ros-sports/rqt_image_overlay/blob/99ef409fcd6fd76a0a6760ab33bb5d43d190a525/rqt_image_overlay/test/test_list_image_topics.cpp#L61), an error is logged, but the tests pass.

Error log: https://build.ros2.org/job/Rdev__rqt_image_overlay__ubuntu_jammy_amd64/8/console

```
14:14:40 test 9
14:14:40       Start  9: test_list_image_topics
14:14:40 
14:14:40 9: Test command: /usr/bin/python3.9 "-u" "/opt/ros/rolling/share/ament_cmake_test/cmake/run_test.py" "/tmp/ws/test_results/rqt_image_overlay/test_list_image_topics.gtest.xml" "--package-name" "rqt_image_overlay" "--output-file" "/tmp/ws/build_isolated/rqt_image_overlay/ament_cmake_gtest/test_list_image_topics.txt" "--command" "/tmp/ws/build_isolated/rqt_image_overlay/test/test_list_image_topics" "--gtest_output=xml:/tmp/ws/test_results/rqt_image_overlay/test_list_image_topics.gtest.xml"
14:14:40 9: Test timeout computed to be: 60
14:14:40 9: -- run_test.py: invoking following command in '/tmp/ws/build_isolated/rqt_image_overlay/test':
14:14:40 9:  - /tmp/ws/build_isolated/rqt_image_overlay/test/test_list_image_topics --gtest_output=xml:/tmp/ws/test_results/rqt_image_overlay/test_list_image_topics.gtest.xml
14:14:40 9: Running main() from /opt/ros/rolling/src/gtest_vendor/src/gtest_main.cc
14:14:40 9: [==========] Running 5 tests from 1 test suite.
14:14:40 9: [----------] Global test environment set-up.
14:14:40 9: [----------] 5 tests from TestListImageTopics
14:14:40 9: [ RUN      ] TestListImageTopics.TestNone
14:14:40 9: [       OK ] TestListImageTopics.TestNone (17 ms)
14:14:40 9: [ RUN      ] TestListImageTopics.TestOne
14:14:40 9: [ERROR] [1646198080.676583588] [test_node]: SubscriberPlugin::subscribeImpl with five arguments has not been overridden
14:14:40 9: [       OK ] TestListImageTopics.TestOne (24 ms)
14:14:40 9: [ RUN      ] TestListImageTopics.TestThree
14:14:40 9: [ERROR] [1646198080.693634940] [test_node]: SubscriberPlugin::subscribeImpl with five arguments has not been overridden
14:14:40 9: [ERROR] [1646198080.694140700] [test_node]: SubscriberPlugin::subscribeImpl with five arguments has not been overridden
14:14:40 9: [ERROR] [1646198080.694648061] [test_node]: SubscriberPlugin::subscribeImpl with five arguments has not been overridden
14:14:40 9: [       OK ] TestListImageTopics.TestThree (19 ms)
14:14:40 9: [ RUN      ] TestListImageTopics.TestOnlyTheoraTransport
14:14:40 9: [ERROR] [1646198080.711868997] [test_node]: SubscriberPlugin::subscribeImpl with five arguments has not been overridden
14:14:40 9: [       OK ] TestListImageTopics.TestOnlyTheoraTransport (16 ms)
14:14:40 9: [ RUN      ] TestListImageTopics.TestFakeTheoraTransport
14:14:40 9: [ERROR] [1646198080.727870818] [test_node]: SubscriberPlugin::subscribeImpl with five arguments has not been overridden
14:14:40 9: [       OK ] TestListImageTopics.TestFakeTheoraTransport (16 ms)
14:14:40 9: [----------] 5 tests from TestListImageTopics (92 ms total)
14:14:40 9: 
14:14:40 9: [----------] Global test environment tear-down
14:14:40 9: [==========] 5 tests from 1 test suite ran. (95 ms total)
14:14:40 9: [  PASSED  ] 5 tests.
14:14:40 9: -- run_test.py: return code 0
14:14:40 9: -- run_test.py: inject classname prefix into gtest result file '/tmp/ws/test_results/rqt_image_overlay/test_list_image_topics.gtest.xml'
14:14:40 9: -- run_test.py: verify result file '/tmp/ws/test_results/rqt_image_overlay/test_list_image_topics.gtest.xml'
14:14:40  9/11 Test  #9: test_list_image_topics ...........   Passed    0.23 sec
```

--- a/rqt_image_overlay/package.xml
+++ b/rqt_image_overlay/package.xml
@@ -20,7 +20,7 @@
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
   <test_depend>ament_cmake_gtest</test_depend>
-  <test_depend>theora_image_transport</test_depend>
+  <test_depend>compressed_image_transport</test_depend>
   <test_depend>std_msgs</test_depend>
 
   <export>

--- a/rqt_image_overlay/test/CMakeLists.txt
+++ b/rqt_image_overlay/test/CMakeLists.txt
@@ -1,7 +1,8 @@
 # Build test_list_image_topics
 find_package(rclcpp REQUIRED)
-find_package(theora_image_transport REQUIRED)
+find_package(compressed_image_transport REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
 
 ament_add_gtest(test_list_image_topics
   test_list_image_topics.cpp)
@@ -12,8 +13,9 @@ target_link_libraries(test_list_image_topics
 
 ament_target_dependencies(test_list_image_topics
   rclcpp
-  theora_image_transport
-  std_msgs)
+  compressed_image_transport
+  std_msgs
+  sensor_msgs)
 
 # Build test_msg_storage
 ament_add_gtest(test_msg_storage

--- a/rqt_image_overlay/test/test_list_image_topics.cpp
+++ b/rqt_image_overlay/test/test_list_image_topics.cpp
@@ -19,11 +19,11 @@
 #include "rclcpp/node.hpp"
 #include "rclcpp/rclcpp.hpp"
 #include "image_transport/image_transport.hpp"
-#include "theora_image_transport/msg/packet.hpp"
+#include "sensor_msgs/msg/compressed_image.hpp"
 #include "std_msgs/msg/string.hpp"
 
-// In this test, there's two types of transports (ie. "raw" and "theora") available.
-// The "theora" transport is declared as a test dependency so we can write tests for
+// In this test, there's two types of transports (ie. "raw" and "compressed") available.
+// The "compressed" transport is declared as a test dependency so we can write tests for
 // image_transport plugins.
 class TestListImageTopics : public ::testing::Test
 {
@@ -71,7 +71,7 @@ TEST_F(TestListImageTopics, TestOne)
   auto imageTopics = rqt_image_overlay::ListImageTopics(*node);
   ASSERT_EQ(imageTopics.size(), 2u);
   EXPECT_TRUE(checkContains(imageTopics, rqt_image_overlay::ImageTopic{"/test_topic", "raw"}));
-  EXPECT_TRUE(checkContains(imageTopics, rqt_image_overlay::ImageTopic{"/test_topic", "theora"}));
+  EXPECT_TRUE(checkContains(imageTopics, rqt_image_overlay::ImageTopic{"/test_topic", "compressed"}));
 }
 
 TEST_F(TestListImageTopics, TestThree)
@@ -95,7 +95,7 @@ TEST_F(TestListImageTopics, TestThree)
   EXPECT_TRUE(
     checkContains(
       imageTopics,
-      rqt_image_overlay::ImageTopic{"/test_ns1/test_topic1", "theora"}));
+      rqt_image_overlay::ImageTopic{"/test_ns1/test_topic1", "compressed"}));
   EXPECT_TRUE(
     checkContains(
       imageTopics,
@@ -103,7 +103,7 @@ TEST_F(TestListImageTopics, TestThree)
   EXPECT_TRUE(
     checkContains(
       imageTopics,
-      rqt_image_overlay::ImageTopic{"/test_ns2/test_topic2", "theora"}));
+      rqt_image_overlay::ImageTopic{"/test_ns2/test_topic2", "compressed"}));
   EXPECT_TRUE(
     checkContains(
       imageTopics,
@@ -111,15 +111,15 @@ TEST_F(TestListImageTopics, TestThree)
   EXPECT_TRUE(
     checkContains(
       imageTopics,
-      rqt_image_overlay::ImageTopic{"/test_ns3/test_topic3", "theora"}));
+      rqt_image_overlay::ImageTopic{"/test_ns3/test_topic3", "compressed"}));
 }
 
-TEST_F(TestListImageTopics, TestOnlyTheoraTransport)
+TEST_F(TestListImageTopics, TestOnlyCompressedTransport)
 {
   auto node = std::make_shared<rclcpp::Node>("test_node");
 
   auto publisher =
-    node->create_publisher<theora_image_transport::msg::Packet>("/test_topic/theora", 1);
+    node->create_publisher<sensor_msgs::msg::CompressedImage>("/test_topic/compressed", 1);
 
   // Give a chance for the topic to be picked up
   rclcpp::sleep_for(std::chrono::milliseconds(10));
@@ -130,17 +130,17 @@ TEST_F(TestListImageTopics, TestOnlyTheoraTransport)
   EXPECT_TRUE(
     checkContains(
       imageTopics,
-      rqt_image_overlay::ImageTopic{"/test_topic", "theora"}));
+      rqt_image_overlay::ImageTopic{"/test_topic", "compressed"}));
 }
 
-TEST_F(TestListImageTopics, TestFakeTheoraTransport)
+TEST_F(TestListImageTopics, TestFakeCompressedTransport)
 {
   // In this example, we test a case where the topic name ends with a transport type, but the msg
-  // type is not theora_image_transport::msg::Packet.
+  // type is not sensor_msgs::msg::CompressedImage.
   auto node = std::make_shared<rclcpp::Node>("test_node");
 
   auto publisher =
-    node->create_publisher<std_msgs::msg::String>("/test_topic/theora", 1);
+    node->create_publisher<std_msgs::msg::String>("/test_topic/compressed", 1);
 
   // Give a chance for the topic to be picked up
   rclcpp::sleep_for(std::chrono::milliseconds(10));

--- a/rqt_image_overlay/test/test_list_image_topics.cpp
+++ b/rqt_image_overlay/test/test_list_image_topics.cpp
@@ -71,7 +71,8 @@ TEST_F(TestListImageTopics, TestOne)
   auto imageTopics = rqt_image_overlay::ListImageTopics(*node);
   ASSERT_EQ(imageTopics.size(), 2u);
   EXPECT_TRUE(checkContains(imageTopics, rqt_image_overlay::ImageTopic{"/test_topic", "raw"}));
-  EXPECT_TRUE(checkContains(imageTopics, rqt_image_overlay::ImageTopic{"/test_topic", "compressed"}));
+  EXPECT_TRUE(
+    checkContains(imageTopics, rqt_image_overlay::ImageTopic{"/test_topic", "compressed"}));
 }
 
 TEST_F(TestListImageTopics, TestThree)

